doc/stdlib/dune

```diff
@@ -0,0 +1,17 @@
+; This is an ad-hoc rule to easy the migration, it should be handled
+; natively by Dune in the future.
+(rule
+  (targets html)
+  (deps
+   (source_tree %{project_root}/theories)
+   (source_tree %{project_root}/plugins)
+   (:header %{project_root}/doc/common/styles/html/coqremote/header.html)
+   (:footer %{project_root}/doc/common/styles/html/coqremote/footer.html))
```

Comment:
Doesn't coqdoc read glob files or some such thing?

Reply:
Good point, on the current setup they will be there as part of the artifacts but indeed we should add the package dependency on.

When native support comes the right rule will be generated tho.